charts/incubator/misskey/values.yaml

```diff
@@ -0,0 +1,163 @@
+image:
+  repository: misskey/misskey
+  pullPolicy: IfNotPresent
+  tag: latest
```

Comment:
we recommend not using latest tags if versioning scheme is created.

for example use this until our mirror is created if you made the pr in the `containers` repo: 

```yaml
image:
  repository: misskey/misskey
  tag: 12.119.0@sha256:a7beecb59b5089068463ca3847ac34dcb8445108da75ed8f56822f670053aa31
  pullPolicy: IfNotPresent
```

Reply:
just copy this into `values.yaml` and replace your image section.

`@sha256:dkcdkmcdkmdc` is **required**.

Reply:
PR for container https://github.com/truecharts/containers/pull/10812. Will update to tccr.io when merged.